build_tools/buildkite/cmake/linux/x86-64/benchmark.yml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+steps:
+  - label: "Build"
+    commands:
+      - "docker run --user=$(id -u):$(id -g) --volume=\\${HOME?}:\\${HOME?} --volume=/etc/passwd:/etc/passwd:ro --volume=/etc/group:/etc/group:ro --volume=\\$PWD:\\$IREE_DOCKER_WORKDIR --workdir=\\$IREE_DOCKER_WORKDIR --rm gcr.io/iree-oss/frontends@sha256:e0a5d9a71cc7f087f4caaea0cc9b9a86ab4765b217970772780e446b4fdc65b0 build_tools/cmake/build_linux_benchmark.sh"
```

Comment:
I recently updated our Docker images in https://github.com/google/iree/commit/0eced24eef88bfad81b0b95997775d47dda24601, so check that you're using the latest versions (this should be `2ef3b3670808cdaa2a483dddf94fc58d50486148eb73f9f87e8a9d08a628d39a` at time of writing)

Reply:
Thanks for the head up. Done.